ql/src/test/org/apache/hadoop/hive/ql/io/orc/TestOrcFile.java

```diff
@@ -1818,8 +1818,9 @@ long getTotalMemoryPool() {
     }
 
```

Comment:
I did port all of the TestOrcFile to the vectorized writer in the new file TestVectorOrcFile. Does that address your concerns?